rating/Rating/Rating.html

```diff
@@ -0,0 +1,150 @@
+                                            </div>
+                                            <hr/>
+                                            <div class="typography-padding-cardbadge">
+                                               <div class="typography-p-small typography-padding-cardbadge"> Select Size (UK Size) </div>
+                                               <div class="flex-row col-gap-1rem typography-padding-cardbadget">
+                                                <button type="button" class="button button-outline-primary button-border-medium-width-height border-radius-round">6</button>
+                                                <button type="button" class="button button-outline-primary button-border-medium-width-height border-radius-round">8</button>
+                                                <button type="button" class="button button-outline-primary button-border-medium-width-height border-radius-round">9</button>
+                                                <button type="button" class="button button-outline-primary button-border-medium-width-height border-radius-round">10</button>
+                                               </div>
+                                            </div>
+                                            <div class="flex-row col-gap-2rem typography-padding-top-bottom">
+                                                <button type="button" class="icon_button button-outline-primary flex-row button-onhover-fillbackground"> 
+                                                    <i class="material-icons">shopping_cart</i> 
+                                                    <label class="typography-fontweight-semibold"> Shopping Cart </label>
+                                                </button> 
+                                                <button type="button" class="icon_button button-outline-primary flex-row button-onhover-fillbackground"> 
+                                                    <i class="material-icons">shopping_basket</i> 
+                                                    <label class="typography-fontweight-semibold"> Add to wishlist </label>
+                                                </button> 
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                <!-- closed container component  -->
+                        </div>
+                    </div>
+     <!-- container component aside -->
+            
+                    <aside class="component-sidebar-container margin-top-right-bottom-left">
+                        <div class="sidebar-onthispage-content"> 
+                        <br/><br/>
+                        <ul>
+
+                        </ul>
+                        </div>
+                    </aside>
+      <!--  component aside closed-->
```

Comment:
here is an empty section, either fill it or remove it.

Reply:
For the page component, I'm using a grid. When I remove this, the "display component container size increased (using width: webkit-fill-available)" .I don't want to increase my component size. 